test/selenium/sessions_ui_test.js

```diff
@@ -255,6 +288,34 @@ describe('sessions ui', function() {
                            5000);
       });
     });
-  });
 
+    describe('tests changing views', function(){
+
+        before(function() {
+          driver.navigate().refresh();
+          signInPage = new SignInPage(driver);
+
+        });
+        beforeEach(function(){
+          return driver.wait(webdriver.until.titleIs('FoxBox'), 5000).then(
+            function() {
+          return signInPage.openPage();
+          }).then(function(signedInPageView) {
+             signedInPage = signedInPageView;
+             return signedInPage;
```

Comment:
I had the return in the previous line (304) but travis was complaining: Did you mean to return a conditional instead of an assignment?